Acr.XamForms.SignaturePad.iOS/SignatureServiceView.cs

```diff
@@ -1,62 +1,53 @@
 using System;
 using CoreGraphics;
 using UIKit;
-using NativeView = global::SignaturePad.SignaturePadView;
+using NativeView = SignaturePad.iOS.SignaturePadView;
 
 
-namespace Acr.XamForms.SignaturePad.iOS {
+namespace SignaturePad.iOS
+{
+	public class SignatureServiceView : UIView
+	{
+		public NativeView Signature { get; set; }
+		public UIButton SaveButton { get; private set; }
+		public UIButton CancelButton { get; private set; }
 
-    public class SignatureServiceView : UIView {
 
-        public NativeView Signature { get; set; }
-        public UIButton SaveButton { get; private set; }
-        public UIButton CancelButton { get; private set; }
+		public SignatureServiceView()
+		{
+			//BackgroundColor = UIColor.White;
+			this.SaveButton = UIButton.FromType(UIButtonType.RoundedRect);
+			this.CancelButton = UIButton.FromType(UIButtonType.RoundedRect);
+			this.Frame = (CGRect)UIScreen.MainScreen.ApplicationFrame;
 
+			this.Signature = new NativeView();
+			this.TranslatesAutoresizingMaskIntoConstraints = false;
 
-        public SignatureServiceView() {
-            //BackgroundColor = UIColor.White;
-            this.SaveButton = UIButton.FromType(UIButtonType.RoundedRect);
-            this.CancelButton = UIButton.FromType(UIButtonType.RoundedRect);
-            this.Frame = (CGRect)UIScreen.MainScreen.ApplicationFrame;
+			this.AddSubview(this.Signature);
+			this.AddSubview(this.SaveButton);
+			this.AddSubview(this.CancelButton);
+		}
 
-            this.Signature = new NativeView();
-            this.TranslatesAutoresizingMaskIntoConstraints = false;
 
-            this.AddSubview(this.Signature);
-            this.AddSubview(this.SaveButton);
-            this.AddSubview(this.CancelButton);
-        }
+		public override void LayoutSubviews()
+		{
+			var frame = UIScreen.MainScreen.ApplicationFrame;
+			var sbframe = (CGRect)UIApplication.SharedApplication.StatusBarFrame;
+			var portrait = UIApplication.SharedApplication.StatusBarOrientation.HasFlag(UIInterfaceOrientation.Portrait);
 
 
-        public override void LayoutSubviews() {
-            var frame = (CGRect)this.Frame;
-            var sbframe = (CGRect)UIApplication.SharedApplication.StatusBarFrame;
-            var portrait = UIApplication.SharedApplication.StatusBarOrientation.HasFlag(UIInterfaceOrientation.Portrait);
+			this.Frame = new CGRect(0, frame.Location.Y, frame.Size.Width, frame.Size.Height);
 
-            var width = portrait
-                ? frame.Size.Width
-                : frame.Size.Width - sbframe.Width;
+			this.Signature.Frame = UIDevice.CurrentDevice.UserInterfaceIdiom == UIUserInterfaceIdiom.Phone
+				? new CGRect(10, 10, Frame.Width - 20, Frame.Height - 60)
+				: new CGRect(84, 84, Frame.Width - 168, Frame.Width / 2);
+			if(UIDevice.CurrentDevice.UserInterfaceIdiom == UIUserInterfaceIdiom.Phone && !portrait){
```

Comment:
For some reason, I needed to explicitly added this if statement because when you would have an iPhone and rotate landscape right, from Portrait. It would not cause the proper resize to occur.